models/articles.model.js

```diff
@@ -13,3 +13,20 @@ exports.fetchArticleById = async (article_id) => {
     return res.rows[0];
   }
 };
+
+exports.updateArticle = async (body, article_id) => {
+  const votes = parseInt(body.inc_votes);
+  // console.log(votes, "<-- votes in model");
+  const res = await db.query(
+    `
+  UPDATE articles
+  SET 
+  votes = votes + $1
+  WHERE article_id = $2
+  RETURNING *
```

Comment:
Don't forget your semicolon at the end of an SQL query string

Reply:
Done :)